### PR TITLE
Update of the UL17 catalogue

### PR DESCRIPTION
**Pull request by alesauva**
Adding UL17 signal samples:

- /WminusH_HToGG_WToAll_M125
-  /ZH_HToGG_ZToAll_M125 (11/12 files)
- /WGToLNuG_01J_5f_TuneCP5
- /ZHiggs0L1ToGG_M125_TuneCP5
- /ggZH_HToGG_ZToLL_M124_TuneCP5

### CODE REVIEW

File: MetaData/data/Era2017_legacy_v1/datasets.json [hunk 1]
```diff
@@ -3851,6 +3851,96 @@
...
+                "name": "/store/user/alesauva/flashgg/UL2017_4/10_6_4/WminusH_HToGG_WToAll_M125_TuneCP5_13TeV-powheg-pythia8/UL2017_4-10_6_4-v0-RunIISummer19UL17MiniAOD-106X_mc2017_realistic_v6-v1/200929_090236/0000/myMicroAODOutputFile_10.root", 
+                "nevents": 25000
+            }, 
+            {
+                "name": "/store/user/alesauva/flashgg/UL2017_4/10_6_4/WminusH_HToGG_WToAll_M125_TuneCP5_13TeV-powheg-pythia8/UL2017_4-10_6_4-v0-RunIISummer19UL17MiniAOD-106X_mc2017_realistic_v6-v1/200929_090236/0000/myMicroAODOutputFile_13.root", 
+                "nevents": 20700
+            }
+        ], 
+        "parent_n_units": null
+    },
```
Comment (youyingli) [inline MetaData/data/Era2017_legacy_v1/datasets.json]: You need to run the following command to add the status and weight for WH

`fggManageSample.py -C <Campaign> check -S <Dataset>`

Comment (alesauva) [reply MetaData/data/Era2017_legacy_v1/datasets.json]: Thanks for your checks !

**Comment by youyingli** [comment]
Do you confirm this PR also includes PR #1229 ? If so, please edit the initial comment in this PR. I will merge it and close PR #1229 .

**Comment by alesauva** [comment]
Yes this catalogue is up to date and also include the PR #1229, I have edited the comments.

**Pull request merged**